libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.kt

```diff
@@ -355,7 +355,7 @@ open class PushService {
 
     private fun getRestClient(
         account: UserAccount
-    ) = SalesforceSDKManager.getInstance().clientManager?.let { clientManager ->
+    ) = SalesforceSDKManager.getInstance().clientManager.let { clientManager ->
```

Comment:
Everything in this source is inspector cleanup for existing trivial issues and those introduced by the Kotlin conversion.